apps/meteor/app/lib/server/functions/deleteUser.ts

```diff
@@ -61,14 +62,27 @@ export async function deleteUser(userId: string, confirmRelinquish = false): Pro
 
 		Subscriptions.removeByUserId(userId); // Remove user subscriptions
 
+		if (user.roles.find((r: string) => r === 'livechat-agent')) {
+			// Remove user as livechat agent
+			LivechatDepartmentAgents.removeByAgentId(userId);
+		}
+
+		if (user.roles.find((r: string) => r === 'livechat-monitor')) {
+			// Remove user as Unit Monitor
+			LivechatUnitMonitors.removeByMonitorId(userId);
+		}
+
+		// Remove user as contact manager for all visitors
+		LivechatVisitors.removeContactManagerByUsername(user.username);
```

Comment:
This should happen only when the user is agent/monitor/manager too 👀

Reply:
done!